user-contrib/Ltac2/g_ltac2.mlg

```diff
@@ -91,7 +91,7 @@ let test_ampersand_ident =
 
 let test_dollar_ident =
   entry_of_lookahead "test_dollar_ident" begin
-    lk_kw "$" >> lk_ident
+    lk_kw "$" >> lk_ident >> check_no_space
```

Comment:
Should you do the same to `lk_ident_or_anti`? It's unlikely to actually cause problems but just for uniformity.

Reply:
Probably. Will fix.

Reply:
Done.